core/src/io/anuke/mindustry/ui/fragments/PlacementFragment.java

```diff
@@ -170,9 +253,21 @@ public void build(Group parent){
                             lastGround = false;
 
                             topTable.table(header -> {
+                                String keyCombo = new String();
+                                if(!mobile){
+                                    keyCombo = " [accent][" + Core.keybinds.get(blockSelect[currentCategory.ordinal()]).key.toString();
```

Comment:
Instead of making it highlighted, I would make this text `"\n[lightgray]Key: " + <etc>` in order to clear up confusion on *what it means* and jump out less. I would also put this into a bundle entry and format it, e.g. `\n[lightgray]Key: [{0}`
(with an extra escaped bracket just in case.)

Reply:
Will fix.

Reply:
I've made suggested changes, as well as added a "Show Block Select Keys" graphics toggle for people who have keys memorized, or want less clutter, or are using controller.